packages/web/app/src/lib/preflight/components/EditorTitle.tsx

```diff
@@ -0,0 +1,10 @@
+import { ReactNode } from 'react';
+import { cn } from '@/lib/utils';
+
+export function EditorTitle(props: { children: ReactNode; className?: string }) {
```

Comment:
Can't we just do `shared-components.ts` instead of creating 10 files where each contains 5LOC.
If we ever need to rename the component, we will have to rename a file, we will loose a diff if not done carefully etc.

I see only downsides.